src/pages/Search/Search.js

```diff
@@ -1,19 +1,164 @@
 import React, {Component} from 'react';
-import form from 'react-bootstrap'
+import Select from 'react-select';
+import './Search.css';
+
+const options1 = [
+  {value: 'bob', label: 'Bob'},
+  {value: 'joe', label: 'Joe'},
+  {value: 'billy', label: 'Billy'}
+]
+
+const options2 = [
+  {value: 'blue', label: 'Blue'},
+  {value: 'yellow', label: 'Yellow'},
+  {value: 'red', label: 'Red'}
+]
+
+const options3 = [
+  {value: 'blue', label: 'Blue'},
+  {value: 'yellow', label: 'Yellow'},
+  {value: 'red', label: 'Red'}
+]
+
+const options4 = [
+  {value: 'thirty', label: '30 mins'},
+  {value: 'forty_five', label: '45 mins'},
+  {value: 'sixty', label: '60 mins'}  
+]
+
+{/*
+should this NOT be a dropdown? 
+i.e. can the clinician put in ANYTHING?
+do clinicians ever book an extremely large number of sessions at once?
+  or is the max they ever do like 10 or something?
```

Comment:
Some clarification: The person booking is called a CSA; they book for clinicians. It would be rare for there to be more than 5 clinicians for a single session.